### ISSUE TITLE
failing R CMD CHECK

### ISSUE THREAD

**Issue by mdingemanse** [elpaco-escience/talkr#128]
My latest commit (a simple readme.md update) results in a failing check that I don't quite know how to diagnose... @PabRod could you have a look? 
https://github.com/elpaco-escience/talkr/commit/ccec0fa75919049b7aad832569289f3b2ae7c55a

**Comment by PabRod** [comment]
I wouldn't give this too much relevance. It failed only in one OS (Ubuntu-latest), and the changes are only in the docs. My guess: not our fault.

Anyways, I am rerunning the action as we speak ;)

**Comment by PabRod** [comment]
And... my guess was wrong.

It seems to be related to [this](https://github.com/elpaco-escience/talkr/pull/125#issuecomment-2382626301), _testthat_ snapshots OS-dependency disapointing us one more time.

**Comment by mdingemanse** [comment]
how do we fix it? If the snapshots are only creating problems I am not sure they are worth the bother TBH

right now this is making me more reluctant to get back to working on talkr; if I'm working towards a new release I would like to do so in the knowledge that needless friction is minimized

**Comment by PabRod** [comment]
I agree. Snapshots sounded like a great idea, but the more we know about them, the less appealing they get.

I'm planning to take a look at this this not later than Friday. Here are the two ways we can go:

1. Only run the snapshot test on a fixed-version OS, not a `latest` (pro: we keep testing, con: makes it a bit annoying to add new snapshots).
3. Remove snapshot tests completely (pro: goodbye to the problem, con: less test coverage).
